lineman/__main__.py

```diff
@@ -5,10 +5,16 @@
 
 Options:
   -h --help                                     show this message and exit
+  -v --version                                  show version.
   -o <output.json> --output=<output.json>       optional output file for results
   -l <log.json> --log=<log.json>                optional log output for infomation related to the run
 
 """
+
+#bring in __version__ from version.py
```

Comment:
Could you delete this comment? I appreciate it being here but it seems like the place for it to be is in the setup.py instead